src/main/java/com/amazonaws/kinesisvideo/producer/KvsRetryStrategyCallbacks.java

```diff
@@ -0,0 +1,16 @@
+package com.amazonaws.kinesisvideo.producer;
+
+/**
+ * This interface holds the retry strategy callback functions.
+ *
+ * NOTE: This should follow the structure defined in PIC's /kvspic-src/src/utils/include/com/amazonaws/kinesis/video/utils/Include.h
+ *
+ */
+public interface KvsRetryStrategyCallbacks {
+
+    void createRetryStrategyFn(KvsRetryStrategy kvsRetryStrategy) throws ProducerException;
```

Comment:
Shouldn't this return something? 

Like, for example, the handle (long)?

Reply:
This aligns with our other callbacks such as `DefaultStreamCallbacks` which return a status in PIC but return void in Java end.